qfieldsync/gui/cloud_transfer_dialog.py

```diff
@@ -179,8 +224,17 @@ def prepare_project_transfer(self):
         self.project_transfer.finished.connect(self.on_transfer_finished)
 
         self.build_files_tree()
-        self.stackedWidget.setCurrentWidget(self.filesPage)
-        self.buttonBox.button(QDialogButtonBox.Apply).setVisible(True)
+
+        if self.download_project:
+            self._on_prefer_cloud_button_clicked()
+            self.on_project_apply_clicked()
+        else:
+            self.build_files_tree()
+            self.stackedWidget.setCurrentWidget(self.filesPage)
+            self.buttonBox.button(QDialogButtonBox.Apply).setVisible(True)
+            self.buttonBox.button(QDialogButtonBox.Apply).setText(
+                self.tr("Download project")
```

Comment:
I am now in a weird situation I just created a new project and it says "Download project"

Reply:
Something must not register properly when you created the project re local_dir? That'd be the only way download appears here.

Reply:
Why not "synchronize project" as neutral?

Reply:
I created a project and set the local dir during creation.

Reply:
I felt it was better to communicate to the user the difference between synching pre-existing local <-> remote vs. populating local (i.e. downloading).